uniffi/src/interface/mod.rs

```diff
@@ -620,6 +659,10 @@ impl Method {
         }
     }
 
+    pub fn error(&self) -> &Option<Error> {
```

Comment:
API nit: it's not clear why `error` is preferred as the name of the function but `throws` as the name of the attribute, and it seems worth being consistent; FWIW my slight preference would be to use `throws` consistently, from IDL definition to struct attribute to public accessor.